package-lock.json

```diff
@@ -1,1192 +1,8 @@
 {
   "name": "developer-guide",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Can you remove this file change, I think you might b e using older version of npm